File: app/src/main/java/com/commit451/gitlab/viewHolder/TagViewHolder.java
package com.commit451.gitlab.viewHolder;

import android.support.v7.widget.RecyclerView;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.TextView;

import com.commit451.easel.Easel;
import com.commit451.gitlab.R;
import com.commit451.gitlab.model.api.Tag;

import butterknife.BindView;
import butterknife.ButterKnife;

/**
 * Label
 */
public class TagViewHolder extends RecyclerView.ViewHolder {

    public static TagViewHolder inflate(ViewGroup parent) {
        View view = LayoutInflater.from(parent.getContext())
                .inflate(R.layout.item_tag, parent, false);
        return new TagViewHolder(view);
    }

    @BindView(R.id.title)
    public TextView title;

    int colorHighlighted;

    public TagViewHolder(View view) {
        super(view);
        ButterKnife.bind(this, view);
        colorHighlighted = Easel.getThemeAttrColor(itemView.getContext(), R.attr.colorControlHighlight);
    }

    public void bind(Tag tag, boolean selected) {
        title.setText(tag.getName());
        if (selected) {
            itemView.setBackgroundColor(colorHighlighted);
        } else {
            itemView.setBackground(null);
        }
    }
}

File: app/src/main/java/com/commit451/gitlab/viewHolder/TagViewHolder.kt
package com.commit451.gitlab.viewHolder

import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.commit451.addendum.recyclerview.bindView
import com.commit451.addendum.recyclerview.context
import com.commit451.addendum.themeAttrColor
import com.commit451.gitlab.R
import com.commit451.gitlab.model.api.Tag

/**
 * Label
 */
class TagViewHolder(view: View) : RecyclerView.ViewHolder(view) {

    companion object {

        fun inflate(parent: ViewGroup): TagViewHolder {
            val view = LayoutInflater.from(parent.context)
                    .inflate(R.layout.item_tag, parent, false)
            return TagViewHolder(view)
        }
    }

    private val title: TextView by bindView(R.id.title)

    private var colorHighlighted: Int = 0

    init {
        colorHighlighted = context.themeAttrColor(R.attr.colorControlHighlight)
    }

    fun bind(tag: Tag, selected: Boolean) {
        title.text = tag.name
        if (selected) {
            itemView.setBackgroundColor(colorHighlighted)
        } else {
            itemView.background = null
        }
    }
}
